modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -7,32 +7,66 @@
 = Using Data Mover for CSI snapshots
 
 :FeatureName: Data Mover for CSI snapshots
-include::snippets/technology-preview.adoc[]
 
-The OADP 1.1.2 Data Mover enables customers to back up container storage interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications from the store if a failure, accidental deletion, or corruption of the cluster occurs. The OADP 1.1.2 Data Mover solution uses the Restic option of VolSync.
+The OADP Data Mover enables customers to back up Container Storage Interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications, using CSI volume snapshots pulled from the object store if a failure, accidental deletion, or corruption of the cluster occurs.
+
+The Data Mover solution uses the Restic option of VolSync.
 
-[NOTE]
-=====
 Data Mover supports backup and restore of CSI volume snapshots only.
 
-Currently, Data Mover does not support Google Cloud Storage (GCS) buckets.
-=====
+In OADP 1.2.x Data Mover VolumeSnapshotBackups (VSBs) and VolumeSnapshotRestores (VSRs) are queued using the VolumeSnapshotMover (VSM). The VSM's performance is improved by specifying a concurrent number of VSBs and VSRs simultaneously in Progress. After all async plugin operations are completed, the backup is marked as completed.
+
```

Comment:
1. In OADP 1.2 Data Mover, `VolumeSnapshotBackup` (VSB) and `VolumenSnapshotRestore` (VSR) resources are queued using the  VolumeSnapshotMover (VSM).  
2. What's the difference between the VSM and the Data Mover? The VSM sounds close to the Data Mover from the description.
3. re: he VSM's performance is improved by specifying a concurrent number of VSBs and VSRs simultaneously in Progress. -- Is there a limit? Is this instead of having the be processed one at a time?  I think "In Progress" is a status. If so, it should be in backticks. Suggestion "VSBs and VSRs that can simultaneously have a status of `In Progress`."

Reply:
@RichardHoch in progress here isn't being used as a status, as far as I know. @PrasadJoshi12 @eemcmullan WDYT?

Reply:
InProgress is a status on VSB and VSR CR.
```
status:
  batchingStatus: Processing
  conditions:
  - lastTransitionTime: "2023-04-11T11:56:46Z"
    message: Reconcile complete
    reason: Complete
    status: "True"
    type: Reconciled
  **phase: InProgress**

```

Reply:
s/completed/complete  (I think that is correct. As in: _When the backup is complete, it is marked as complete_) or is completed a phase name?